### PR TITLE
[ENGG-1592] chore: use color token in rules table

### PR DESCRIPTION
**Pull request by wrongsahil**
Closes issue: 

## 📜 Summary of changes:

CleanShot 2024-04-10 at 01 45 41

## ✅ Checklist:

- [ ] Make sure linting and unit tests pass.
- [ ] No install/build warnings introduced.
- [ ] Verified UI in browser.
- [ ] For UI changes, added/updated analytics events (if applicable).
- [ ] For changes in extension's code, manually tested in Chrome and Firefox.
- [ ] For changes in extension's code, both MV2 and MV3 are covered.
- [ ] Added/updated unit tests for this change.
- [ ] Raised pull request to update corresponding documentation (if already exists).

### CODE REVIEW

File: app/src/features/rules/container.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.rules-feature-content-container {
```
Comment (rohanmathur91) [inline app/src/features/rules/container.scss:1]: Rename this to `rules-content-container`

File: app/src/features/rules/container.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  background-color: var(--requestly-color-background);
+  flex-grow: 1;
+  height: 100%;
+  overflow: auto !important;
+}
+
+.rules-feature-container {
```
Comment (rohanmathur91) [inline app/src/features/rules/container.scss:8]: `.rules-feature-container ` -> `.rules-container `

File: app/src/features/rules/screens/rulesList/components/RulesList/RulesList.tsx [hunk 3]
```diff
@@ -74,6 +74,11 @@ const RulesList: React.FC<Props> = () => {
           <>
             <div className="rq-rules-list-container">
               {isFeatureLimiterOn && user.isLimitReached ? <FeatureLimiterBanner /> : null}
+
+              {/* TODO: Temp Breadcrumb */}
+              <div className="rq-rules-table-breadcrumb">
+                <span className="breadcrumb-1">Rules</span> {" > "} <span className="breadcrumb-2">All</span>
```
Comment (rohanmathur91) [inline app/src/features/rules/screens/rulesList/components/RulesList/RulesList.tsx:80]: Let's hide this till its not completed

File: app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/rulesTable.css [hunk 4]
```diff
@@ -3,65 +3,62 @@
...
-}
-
 .rule-more-actions-dropdown .ant-dropdown-menu {
   padding: 4px;
-  background: var(--surface-1);
-  border: 1px solid var(--surface-2);
+  background: var(--requestly-color-surface-0);
+  border: 1px solid var(--requestly-color-surface-1);
   border-radius: var(--border-radius-sm);
   box-shadow: 0px 8px 12px 0px #0000003d;
```
Comment (rohanmathur91) [inline app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/rulesTable.css:63]: Please update the box-shadow color too with color variable

**Review by rohanmathur91 — APPROVED** [review]

File: app/src/features/rules/screens/rulesList/components/RulesList/RulesList.tsx (hunk 3 above)
Comment (wrongsahil) [reply app/src/features/rules/screens/rulesList/components/RulesList/RulesList.tsx:80]: Looks weird without this. So keeping this for now

**Pull request merged**